plugins/modules/mysql_info.py

```diff
@@ -255,7 +257,8 @@
       "plugin": "mysql_native_password",
       "priv": "db1.*:SELECT/db2.*:SELECT",
       "resource_limits": { "MAX_USER_CONNECTIONS": 100 },
-      "tls_requires": { "SSL": null } }
+      "tls_requires": { "SSL": null } },
+      "locked": false
   version_added: '3.8.0'
```

Comment:
```suggestion
      "tls_requires": { "SSL": null },
      "locked": false }
  version_added: '3.8.0'
```

Reply:
would it be correct?

Reply:
@Andersson007 thanks a lot for the review! I don't believe I missed that.